### PR TITLE
Change the hierarchy issue messages

### PR DESCRIPTION
**Pull request by RealFriesi**
### Type

- [ ] Bugfix
- [x] Enhancement
- [ ] New feature

### Description of the changes
Add custom messages for adding and removing roles. This removed the verb variable which makes it difficult to translate the messages.

### CODE REVIEW

**Review by mikeshardmind — CHANGES_REQUESTED** [review]
Thanks! Changes like this to support i18n better are good to have.

If you can toss a brief [changelog entry](https://github.com/Cog-Creators/Red-DiscordBot/blob/V3/develop/.github/CONTRIBUTING.md#47-using-towncrier) into this PR, we'll get it merged in.

If anything is unclear about the process there, please ask.

File: redbot/cogs/admin/admin.py [hunk 1]
```diff
@@ -19,15 +19,29 @@
...
+    "I tried to add {role.name} to {member.display_name} but that role"
     " is higher than my highest role in the Discord hierarchy so I was"
     " unable to successfully add it. Please give me a higher role and "
     "try again."
 )
 
-USER_HIERARCHY_ISSUE = _(
-    "I tried to {verb} {role.name} to {member.display_name} but that role"
+HIERARCHY_ISSUE_REMOVE = _(
+    "I tried to remove {role.name} to {member.display_name} but that role"
```
Comment (Flame442) [inline redbot/cogs/admin/admin.py]: This phrase should be "I tried to remove {role.name} **from** ..."

File: redbot/cogs/admin/admin.py [hunk 2]
```diff
@@ -19,15 +19,29 @@
...
+    " is higher than my highest role in the Discord hierarchy so I was"
+    " unable to successfully add it. Please give me a higher role and "
```
Comment (Flame442) [inline redbot/cogs/admin/admin.py]: This phrase should be "unable to successfully **remove** it."

File: redbot/cogs/admin/admin.py [hunk 3]
```diff
@@ -19,15 +19,29 @@
...
+
+USER_HIERARCHY_ISSUE_ADD = _(
+    "I tried to add {role.name} to {member.display_name} but that role"
+    " is higher than your highest role in the Discord hierarchy so I was"
+    " unable to successfully add it. Please get a higher role and "
+    "try again."
+)
+
+USER_HIERARCHY_ISSUE_REMOVE = _(
+    "I tried to remove {role.name} to {member.display_name} but that role"
```
Comment (Flame442) [inline redbot/cogs/admin/admin.py]: Same issue as above, "**from**" and "**remove**".

**Review by Flame442 — CHANGES_REQUESTED** [review]
I know that these issues are not a result of your changes, but they should still be fixed regardless. Thank you for contributing.

**Comment by RealFriesi** [comment]
I think I did everything right now :D

**Review by mikeshardmind — APPROVED** [review]
Looks solid, thanks!

**Pull request merged**